pkg/sdn/plugin/proxy.go

```diff
@@ -205,20 +125,7 @@ EndpointLoop:
 		for _, ss := range ep.Subsets {
 			for _, addr := range ss.Addresses {
 				IP := net.ParseIP(addr.IP)
-				if ni.ServiceNetwork.Contains(IP) {
-					glog.Warningf("Service '%s' in namespace '%s' has an Endpoint inside the service network (%s)", ep.ObjectMeta.Name, ns, addr.IP)
-					continue EndpointLoop
-				} else if ni.ClusterNetwork.Contains(IP) {
-					podInfo, ok := proxy.getTrackedPod(addr.IP)
-					if !ok {
-						glog.Warningf("Service '%s' in namespace '%s' has an Endpoint pointing to non-existent pod (%s)", ep.ObjectMeta.Name, ns, addr.IP)
-						continue EndpointLoop
-					}
-					if podInfo.ObjectMeta.Namespace != ns {
-						glog.Warningf("Service '%s' in namespace '%s' has an Endpoint pointing to pod %s in namespace '%s'", ep.ObjectMeta.Name, ns, addr.IP, podInfo.ObjectMeta.Namespace)
-						continue EndpointLoop
-					}
-				} else {
+				if !ni.ClusterNetwork.Contains(IP) && !ni.ServiceNetwork.Contains(IP) {
```

Comment:
yeah, not sure what the right thing to do is, can you spawn a follow-up issue to track?